From dff0bcbeefa77c1e9d006041ca247513110e1afb Mon Sep 17 00:00:00 2001
From: Avery Pennarun <apenwarr@gmail.com>
Date: Mon, 12 May 2014 20:29:55 -0400
Subject: [PATCH 1/2] sysfs: register /sys/class/ieee80211/phy attributes even
 on older kernels.

Otherwise 'iw phy phy0 set antenna ...' gives "no such file or directory."
---
 net/wireless/sysfs.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/net/wireless/sysfs.c b/net/wireless/sysfs.c
index 7d3162b..c5d39ed 100644
--- a/net/wireless/sysfs.c
+++ b/net/wireless/sysfs.c
@@ -170,9 +170,7 @@ struct class ieee80211_class = {
 
 int wiphy_sysfs_init(void)
 {
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,11,0)
 	init_ieee80211_attrs();
-#endif
 	return class_register(&ieee80211_class);
 }
 
-- 
1.9.1.423.g4596e3a

